src/control/Control.cpp

```diff
@@ -2468,6 +2470,26 @@ void Control::applyPreferredLanguage() {
 #endif
 }
 
+void Control::initButtonTool() {
+    ButtonConfig* cfg = settings->getEraserButtonConfig();
+    cfg->initButton(this->toolHandler, Button::eraser);
+
+    cfg = settings->getStylusButton1Config();
+    cfg->initButton(this->toolHandler, Button::one);
```

Comment:
Please make this `Button::stylusOne` and so on.

Reply:
Agreed. That's the better wording :+1:

Reply:
Done :heavy_check_mark: